### DISCUSSION TITLE
Leaderboard of the month tracking - add a hook similar to AotW

### DISCUSSION THREAD

**Discussion by kdecks** [RetroAchievements/RAWeb#752] [Ideas]
The purpose of this idea is so that Leaderboard of the Month can become a more official part of RA, and so we can track stats, possibly in the future give unique rewards, and report current leaders and winners in Discord by @meleu's bot.

I/we can elaborate on this ideas, as this is rather vague.

**Comment by Ben-Ghazi** [comment]
To add to this I was thinking we could make the bot post to #mastery-news every time there's a new entry or when someone improves their score and their current rank since it would be nice to see that channel used for events like it used to be. How does that sound?

I do see a potential problem doing this in that it would require a monthly update so the bot knows where to look, maybe we can make the bot detect when there's a new leaderboard when it sees "Leaderboard of the Month - [new month], maybe Meleu can give his insight on the feasibility of that task.

**Comment by meleu** [comment]
Yeah, that info on our discord would be cool indeed. Although the Ben's idea is feasible, I think it doesn't worth the effort to code such makeshift to get an info that will be easy to get in the ~~mythical~~ v2. 

By the way, The focus of this issue is/should be only what the title says, and it'll be done in v2. Let's discuss what RABot can do with this info on the [RABot's issue tracker](https://github.com/RetroAchievements/RABot/issues).
